src/main/java/teammates/ui/pagedata/InstructorFeedbackResponseCommentAjaxPageData.java

```diff
@@ -95,9 +95,9 @@ public boolean isResponseVisibleTo(FeedbackParticipantType participantType, Feed
     public String createEditedCommentDetails(String giverName, String editorName) {
         boolean isGiverAnonymous = Const.DISPLAYED_NAME_FOR_ANONYMOUS_PARTICIPANT.equals(giverName);
         return "From: " + giverName + " ["
-                + TimeHelper.formatDateTimeForSessions(comment.createdAt, sessionTimeZone)
+                + TimeHelper.formatDateTimeForHomePage(comment.createdAt, sessionTimeZone)
```

Comment:
This method is used outside home pages. In this instance, it's used in the results page, where comments are displayed. Would `formatDateTimeForDisplayFull` be a better name?